11-rmarkdown-application-regression.Rmd

```diff
@@ -0,0 +1,217 @@
+---
+title: "Further Application of R Markdown"
+teaching: 45
+exercises: 15
+questions:
+- "How can I use R Markdown to integrate analysis results in my report?"
+objectives:
+- "Introduction to regression with lm()"
+- "Demonstrate how to add a regression line and statistics to a ggplot"
+- "Introduction to the `broom` package for 'tidying' statistical results"
+- "Demonstrate how to include statistical results in a reproducible document"
+source: Rmd
+---
+
+```{r, include=FALSE}
+source("bin/chunk-options.R")
+knitr_fig_path("11-")
+```
+
+# Applications of R Markdown
+
+Now that we've learned the basics of R and R Markdown, we're going to tie some 
+things together by doing a regression on the Gapminder data and displaying our
+results in a nicely formatted document.
+
+Working in a new R Markdown document (File -> New File -> R Markdown Document), 
+we can load the libraries that we are going to use, and load the data in 
+a code chunk:
+
+<pre>
+&#96;&#96;&#96;{r load}
+library(readr)
+library(dplyr)
+library(ggplot2)
+gapminder <- read_csv("data/gapminder_data.csv")
```

Comment:
It looks like you are using both read_csv and read.csv - did we come to a standard? Sam and I are both using read_csv in the R Markdown lesson (10)

Reply:
I think `read_csv` is the consensus. I don't think I used `read.csv` in this lesson, but it is in one of the challenges in lesson 5 - which I think I'm going to change